forge/src/main/java/whocraft/tardis_refined/compat/create/TardisDisplaySource.java

```diff
@@ -0,0 +1,64 @@
+package whocraft.tardis_refined.compat.create;
+
+import com.simibubi.create.api.behaviour.display.DisplaySource;
+import com.simibubi.create.content.redstone.displayLink.DisplayLinkContext;
+import com.simibubi.create.content.redstone.displayLink.target.DisplayTargetStats;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import whocraft.tardis_refined.common.blockentity.device.FlightDetectorBlockEntity;
+import whocraft.tardis_refined.common.tardis.TardisNavLocation;
+import whocraft.tardis_refined.common.capability.tardis.TardisLevelOperator;
+import whocraft.tardis_refined.common.util.MiscHelper;
+import whocraft.tardis_refined.compat.ModCompatChecker;
+import whocraft.tardis_refined.compat.valkyrienskies.VSHelper;
+import whocraft.tardis_refined.constants.ModMessages;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TardisDisplaySource extends DisplaySource {
+
+
+    @Override
+    public List<MutableComponent> provideText(DisplayLinkContext context, DisplayTargetStats stats) {
+
+        BlockEntity blockEntity = context.getSourceBlockEntity();
+
+        ArrayList<MutableComponent> list = new ArrayList<>();
+
+        if (blockEntity instanceof FlightDetectorBlockEntity flightDetectorBlockEntity) {
+            Level level = flightDetectorBlockEntity.getLevel();
+
+            if (level instanceof ServerLevel serverLevel) {
+                boolean isPresent = TardisLevelOperator.get(serverLevel).isPresent();
+                if (isPresent) {
+                    TardisLevelOperator levelOperator = TardisLevelOperator.get(serverLevel).get();
+                    TardisNavLocation currentLoc = levelOperator.getPilotingManager().getCurrentLocation();
+                    if (ModCompatChecker.valkyrienSkies()) {
+                        currentLoc = VSHelper.toWorldLocation(currentLoc);
+                    }
+
+                    list.add(levelOperator.getPilotingManager().isInFlight() ? Component.literal("In Flight: True") : Component.literal("In Flight: False"));
```

Comment:
These should be apart of the translations

Reply:
Same goes for everything below

Reply:
Fixed